retriever/engines/postgres.py

```diff
@@ -141,3 +141,7 @@ def get_connection(self):
         db_encoding = encoding_lookup.get(encoding)
         conn.set_client_encoding(db_encoding)
         return conn
+
+    def to_csv(self):
```

Comment:
we don't need this. When the code is executed, a call to the function that is not overriden will call the function in the main class

Reply:
same applies to mysql,  Just delete  that part